lib/tests-api/actions-builder.js

```diff
@@ -329,6 +329,22 @@ module.exports = class ActionsBuilder {
 
         return this;
     }
+
+    switchToFrame(frameElement) {
+        this._pushAction(this.switchToFrame, function switchToFrame(browser) {
+            return findElement(frameElement, browser).then(browser._wd.frame);
```

Comment:
you can't use private key `_wd` here - you need to expose `frame` from [here](https://github.com/gemini-testing/gemini/blob/master/lib/browser/new-browser.js#L37)